lambda/mcp_server/lambda_functions.py

```diff
@@ -178,19 +178,38 @@ def _is_member(user_groups: List[str], prompt_groups: List[str]) -> bool:
     return bool(set(user_groups) & set(prompt_groups))
 
 
+def _set_can_use(
+    connections: Dict[str, Any], user_id: Optional[str] = None, groups: Optional[List[str]] = None
+) -> Dict[str, Any]:
+    items = connections.get("Items", [])
+    formatted_groups = [f"group:{group}" for group in groups]
```

Comment:
Won't this fail is groups is `None`? Maybe groups should default to an empty list instead of None, then you don't need to do that check a few lines down see if `groups` is truth